Dockerfile

```diff
@@ -8,47 +8,30 @@ RUN \
   set -o errexit -o xtrace; \
   apt-get update; \
   env DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes --no-install-recommends \
-    ca-certificates \
-    clang \
```

Comment:
Most of the packages that I removed were already installed by the new base image. However it's worth noting that the image provides `gcc` instead of `clang`. I don't think that will matter for us, but I'm not sure yet.